app/cafe.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        vaccinated = visitor.get("vaccine")
+        current_date = datetime.date.today()
+        mask_on_face = visitor["wearing_a_mask"]
+        vaccine_exp = vaccinated["expiration_date"] if vaccinated else None
```

Comment:
maybe you could avoid creating additional variables by putting information in  if statement directly for example || if not visitor.get("vaccine") ||

Reply:
depends on teamlead desicion))